examples/index.ts

```diff
@@ -178,13 +178,13 @@ async function grpc() {
         await doAmmStreams(provider)
     }
 
-    if (runLongExamples) {
+    /*if (runLongExamples) {
```

Comment:
uncomment (I'd suggest creating a run config in your IDE that has this environment variable set so we don't accidentally commit code like this)